### DISCUSSION TITLE
The cpu and memory usage are significantly higher

### DISCUSSION THREAD

**Discussion by tomdong2010** [strimzi/strimzi-kafka-operator#8925] [General]
### Bug Description

when restart , the kafka cost cpu and memory is so big ,make my service stuck . And installing kafka is also to long time

### Strimzi version

0.36.0

### Kubernetes version

3.5

### Installation method

yaml

**Comment by ppatierno** [comment]
Hi,
there is pretty much nothing useful to us in the description. We would need more information. At least, the Kafka CR you are using, the environment you are running, the cluster operator log.

**Comment by tomdong2010** [comment]
https://github.com/tomdong2010/operator_log/blob/main/strimzi-cluster-operator-7b4f8f8496-h8vtr_strimzi-cluster-operator.log    you can see ，  this installation process has been going on for 10 minutes   my cpu average  is  4 and memory about average  5Gi  if  i use native installation ,this situation is not appear , i tried many times

**Comment by ppatierno** [comment]
Where is the Kafka CR? Is it minikube? Kubernetes? You should describe the ZK pods, I think you have resources issues and they are not schedulable somehow?

**Comment by tomdong2010** [comment]
in k8s  , but  i  also tried to use native installation  , no such problem

**Comment by ppatierno** [comment]
what does native installation means?
I also asked for other information see my comments above.

**Comment by tomdong2010** [comment]
use helm to install zookeeper and kafka, there is not problem about cpu and memory. and also smooth, this is only my feeling ,maybe my problem . that`s all

**Comment by ppatierno** [comment]
we cannot compare apples and pears ... I would like to help but you are still not providing what I need. See my comments above.

**Comment by scholzj** [comment]
The operator log you shared suggests that the ZooKeeper pods were not ready. But the screenshots shows them as ready. So they do not seem to match with each other.

**Comment by tomdong2010** [comment]
https://github.com/tomdong2010/operator_log/blob/main/zkpod.png this is pod situation , normal   https://github.com/tomdong2010/operator_log/blob/main/pods.png

**Comment by ppatierno** [comment]
Thanks Jakub, that is exactly what I was going to say.

**Comment by ppatierno** [comment]
> https://github.com/tomdong2010/operator_log/blob/main/zkpod.png this is pod situation , normal https://github.com/tomdong2010/operator_log/blob/main/pods.png

So I don't understand now, is it working?

**Comment by tomdong2010** [comment]
I found the operator restart a time just now

**Comment by tomdong2010** [comment]
yep , i used it ok , but But the cpu and memory usage are more obvious . always appear this fail https://github.com/tomdong2010/operator_log/blob/main/fail.png

**Comment by scholzj** [comment]
I think we should probably change this to discussion since there does not seem to be anything suggesting any bug so far and you are not providing the information needed to really do anything about this and your screenshots actually suggest everything works fine one time and doesn't work next time which sounds like an environment issue.

If you think it is consuming too many resources, you should share the information you were asked in the beginning:
* The Kafka custom resource
* Full logs from all components (operator, ZooKeeper nodes, Kafka nodes etc.)
* Explain what you environment is (Strimzi works only on Kubernetes -> so the environment is always Kubernetes. More interesting are things such as Kubernetes distribution used, platform you use, version etc.)
* And you should also explain what does it mean that _it is consuming too many resources_. Which component, how do you measure it, share the charts showing the resource consumption together with the corresponding logs etc.

**Comment by tomdong2010** [comment]
ok ,but above image is a error about Readiness probe failed , this issue always appeared . maybe this is reason.  change this to discussion

**Reply by ppatierno** [reply]
The thing is that for me and Jakub it's really difficult to help if you continue to share just screenshot.
Jakub provided you a more clear list of what we need to be helpful. Without them there is nothing we can do.
I would just go through that list and provide everything we asked for.
